7.0/startup.sh

```diff
@@ -59,6 +70,10 @@ render_tmpl "$HOME_DIR/.acquia/cloudapi.conf"
 # Terminus authentication
 [[ "$SECRET_TERMINUS_TOKEN" ]] && terminus_login
 
+convert_secrets
```

Comment:
Comment on what is going on here with convert_secrets and why. Just like it is commented below for `source ...`